### DISCUSSION TITLE
디스커션 열었습니다.

### DISCUSSION THREAD

**Discussion by Combi153** [woowacourse-teams/2023-trip-draw#351] [기록]
![image](https://github.com/woowacourse-teams/2023-trip-draw/assets/106813090/564d9f97-ed3b-4174-9dc8-94682c342b3f)

카테고리는 위와 같이 만들었습니다.

문서화에 힘쓰는 개발자가 되도록 합시다.

많은 관심 부탁드립니다.

![image](https://github.com/woowacourse-teams/2023-trip-draw/assets/106813090/6e15281c-8659-4539-adb3-6a7e0a1b2b97)

**Comment by greeng00se** [comment]
문서화 잘하자!

**Comment by otter66** [comment]
문서화 잘하자!
근데 카테고리 추가해도 되나요?
"학습" 추가하고싶어요. 
지금 이슈에 그때그때 정리하고 있지만, 나중에 찾기도 힘들고, 그렇다고 블로그에 올리기는 애매한 것들이요! 
새로운 기능 등을 학습한 것을 올릴 수 있으면 좋을 것 같습니다!

**Reply by Combi153** [reply]
추가하세요~~
